storybook/storybook-react/src/Alert/Alert.stories.tsx

```diff
@@ -0,0 +1,108 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import { Alert, Link, Paragraph, UnorderedList } from '@amsterdam/design-system-react'
+import { Meta, StoryObj } from '@storybook/react'
+
+const meta = {
+  title: 'Feedback/Alert',
+  component: Alert,
+  args: {
+    title: 'Let op',
+    closeable: false,
+    icon: false,
+  },
+  argTypes: {
+    type: {
+      control: {
+        type: 'select',
+        labels: { undefined: 'default', error: 'error', success: 'success' },
+      },
+      options: [undefined, 'error', 'success'],
+    },
+    closeable: {
+      control: {
+        type: 'boolean',
+        default: false,
+      },
+    },
+    icon: {
+      control: {
+        type: 'boolean',
+      },
+      if: { arg: 'type', neq: undefined },
+    },
+    onClose: {
+      action: 'onClose',
+    },
+    children: {
+      control: {
+        disable: true,
+      },
+    },
+  },
+} satisfies Meta<typeof Alert>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+export const Default: Story = {
+  args: {
+    type: undefined,
+    children: (
+      <Paragraph>Tijdens Koningsdag zijn alle Stadsloketten gesloten, storingen op formulieren, enz.</Paragraph>
+    ),
+  },
+}
+
+export const InformationWithInlineLink: Story = {
+  args: {
+    type: undefined,
+    children: (
+      <Paragraph>
+        Tijdens Koningsdag zijn <Link variant="inline">alle Stadsloketten</Link> gesloten, storingen op formulieren,
+        enz.
```

Comment:
```suggestion
        Tijdens Koningsdag zijn <Link variant="inline">alle Stadsloketten</Link> gesloten. Ook 14 020 en alle andere telefoonnummers van de gemeente zijn niet bereikbaar.
```

([source](https://www.amsterdam.nl/veelgevraagd/op-welke-feestdagen-is-de-gemeente-gesloten-of-eerder-dicht-92eb6-kp))